mitodl/ocw-hugo-themes
# feat: contact page

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/340

#### What's this PR do?
- Responsive Contact page created
- Used bootstrap to keep css minimum
- Contact form is rendered via Wufoo  (some Wufoo limitations are mentioned [here](https://github.com/mitodl/ocw-hugo-themes/issues/340#issuecomment-1015345555))
- [View Deployed PR](https://ocw-hugo-themes-pr-353--ocw-next.netlify.app/contact/)

#### How should this be manually tested?
- Verify that contact nav button in navbar redirects to contact page
- Verify that page is rendered fine on different screen sizes.
- Verify that all links are redirected to appropriate targets.
- Verify that form is working fine and an entry is created in Wufoo each time user submits a form. 

#### Screenshots (if appropriate)
![contact page laptop](https://user-images.githubusercontent.com/93309234/150523411-f5e92ac6-8b20-46b8-aab6-84a5183887c8.png)

![contact page mobile](https://user-images.githubusercontent.com/93309234/150523467-f5276c41-6515-4146-861c-cd76d4cd33d5.png)
